Dockerfile-alpine.template

```diff
@@ -0,0 +1,69 @@
+FROM openjdk:8-alpine
+
+ENV ROOTDIR /usr/local/
+ENV LD_LIBRARY_PATH /usr/local/lib
+ENV GDAL_VERSION %%GDAL_VERSION%%
+ENV OPENJPEG_VERSION %%OPENJPEG_VERSION%%
+
+ENV ANT_VERSION %%ANT_VERSION%%
+ENV ANT_HOME $ROOTDIR/src/apache-ant-${ANT_VERSION}/
+ENV PATH ${ANT_HOME}/bin:${PATH}
+
+# Load assets
+WORKDIR $ROOTDIR/
+
+ADD http://download.osgeo.org/gdal/${GDAL_VERSION}/gdal-${GDAL_VERSION}.tar.gz $ROOTDIR/src/
```

Comment:
> Because image size matters, using `ADD` to fetch packages from remote URLs is strongly discouraged; you should use `curl` or `wget` instead. That way you can delete the files you no longer need after they’ve been extracted and you don’t have to add another layer in your image.

See: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

I would consider doing this with something like:

```
wget -qO- http://download.osgeo.org/gdal/${GDAL_VERSION}/gdal-${GDAL_VERSION}.tar.gz \
| tar -xzC $ROOTDIR/src/ \
&& cd $ROOTDIR/src/
```